tests/acceptance/course-page/complete-challenge-test.js

```diff
@@ -6,7 +6,7 @@ import { currentURL } from '@ember/test-helpers';
 import { module, test } from 'qunit';
 import { setupAnimationTest } from 'ember-animated/test-support';
 import { setupApplicationTest } from 'codecrafters-frontend/tests/helpers';
-import { signIn } from 'codecrafters-frontend/tests/support/authentication-helpers';
+import { signIn, signInAsStaff } from 'codecrafters-frontend/tests/support/authentication-helpers';
```

Comment:
Unless we're testing staff-specific features, let's try to use `signIn` or `signInAsSubscriber`.